### PR TITLE
Changed query get_jids

### LINKED ISSUE #57524: [BUG] jobs.list_jobs shows "Target": unknown-target when using pgjsonb returner
**Description**

Hello Colleagues, I get an issue related to pgjsonb returner. The problem is: when I try to list jobs with the following command:

salt-run jobs.list_jobs search_function='pkg.update' 

I get "Target": unknown-target

I suppose the issue in jsonb database load column.

CREATE TABLE jids (
       jid varchar(255) NOT NULL primary key,
       **load jsonb NOT NULL**
);

Filename: "/usr/local/lib/python3.6/site-packages/salt/returners/pgjsonb.py"

If you check jsonb schema, it has load column, which has an id property in json, as a name of target.
But other returners have tgt property instead of id, like postgres_local_cache.

if you pass a list of minions to salt -L 'minion1, minion2' the database load column has an id property with one minion and not a list, but it should be a list (array).

Also I checked jid utils file
Filename: /usr/local/lib/python3.6/site-packages/salt/utils/jid.py

it has the format_job_instance method, which also includes tgt, and if it is empty it shows unknown-target, and it shows unknown-target in pgjsonb returner, because it doesn't have tgt property, it uses id.

def format_job_instance(job):
    '''
    Format the job instance correctly
    '''
    ret = {'Function': job.get('fun', 'unknown-function'),
           'Arguments': list(job.get('arg', [])),
           # unlikely but safeguard from invalid returns
           **'Target': job.get('tgt', 'unknown-target'),**
           'Target-type': job.get('tgt_type', 'list'),
           'User': job.get('user', 'root')}

**Setup**
(Please provide relevant configs and/or SLS files (be sure to remove sensitive info).
Filename: "/usr/local/lib/python3.6/site-packages/salt/returners/pgjsonb.py"

**Steps to Reproduce the behavior**
(Include debug logs if possible and relevant)

**Expected behavior**

jobs.list_jobs and jobs.print_job should return Target: [minion1, minion2]. and not unknown-target.

jobs.print_job is also affected and should be changed to something similar

**Screenshots**

**Versions Report**
<details><summary>salt --versions-report</summary>
(Provided by running salt --versions-report. Please also mention any differences in master/minion versions.) 

```
PASTE HERE
```
</details>

**Additional context**

As a workaround I changed get_jids method in file /usr/local/lib/python3.6/site-packages/salt/returners/pgjsonb.py to below and it seems it is working, but I suppose it should be tgt as a target and not id.

def get_jids():
    '''
    Return a list of all job ids
    '''
    with _get_serv(ret=None, commit=True) as cur:

        #sql = '''SELECT jid, load
        #        FROM jids'''
        **sql = '''SELECT j.jid as jid, j.load as load, jsonb_agg(sr.id) as tgt FROM jids j
                join salt_returns sr
                using(jid)
                group by j.jid, j.load'''**

        cur.execute(sql)
        data = cur.fetchall()
        ret = {}
        **for** jid, load, tgt in data:
            **load["tgt"] = tgt**
            ret[jid] = salt.utils.jid.format_jid_instance(jid, load)
        return ret

### PR DESCRIPTION
**Pull request by mazovladimir**
### What does this PR do?
This PR solves issue related to bug 
[BUG] jobs.list_jobs shows "Target": unknown-target when using pgjsonb returner #57524
### What issues does this PR fix or reference?
Fixes: https://github.com/saltstack/salt/issues/57524
Incorret get_jids method for pgjsonb salt_returner
### Previous Behavior
Unknown target when use jobs.list_jobs command 
### New Behavior
Correct Target name when use jobs.list_jobs command
### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltstack.com/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: salt/returners/pgjsonb.py [hunk 1]
```diff
@@ -428,13 +428,16 @@ def get_jids():
     """
     with _get_serv(ret=None, commit=True) as cur:
 
-        sql = """SELECT jid, load
-                FROM jids"""
+        sql = '''SELECT j.jid as jid, j.load as load, jsonb_agg(sr.id) as tgt FROM jids j
```
Comment (twangboy) [inline salt/returners/pgjsonb.py:431]: This is probably going to fail  pre-commit... should be double-quotes `"""`

**Review by twangboy — CHANGES_REQUESTED** [review]

**Comment by twangboy** [comment]
@mazovladimir Looks good. Now we'll need some tests.

**Comment by mazovladimir** [comment]
Colleagues, I suppose I found the issue, no need to replace sql query, there was a missing tgt in minion file. Please see commit above.

**Comment by mazovladimir** [comment]
I suppose this bug related to multiple returners, and should be applied globally to all minions. I tried to add this line in commit and it works perfect.

**Review by dmurphy18 — CHANGES_REQUESTED** [review]
@mazovladimir Are you still interested in this PR ?, if so, please add test cases and address outstanding review comments. If unresponsive, this PR shall be closed after several days

**Comment by Ch3LL** [comment]
Closing due to inactivity. Please let me know if you want me to re-open or please submit a PR against the master branch with the requested changes.

**Pull request closed without merging**